src/components/Docs/components/ForeignCoinsTable.tsx

```diff
@@ -34,13 +48,76 @@ const formatString = (str: string) => {
     .join(" ");
 };
 
+const addTokenToMetaMask = async (token: ForeignCoin & { chainName: string }) => {
+  if (!window.ethereum) {
+    alert("Please install an EVM wallet like MetaMask to add tokens to your wallet");
+    return;
+  }
+
+  try {
+    const wasAdded = await window.ethereum.request({
+      method: "wallet_watchAsset",
+      params: {
+        type: "ERC20",
+        options: {
+          address: token.zrc20_contract_address,
```

Comment:
We could add token icons in the public folder and use them here, it's way nicer to add the tokens with the icons included.